webdriver/tests/bidi/browsing_context/context_created/context_created.py

```diff
@@ -263,3 +263,22 @@ async def on_event(method, data):
     )
 
     remove_listener()
+
+
+@pytest.mark.parametrize("type_hint", ["tab", "window"])
+async def test_existing_context(bidi_session, wait_for_event, wait_for_future_safe, subscribe_events, type_hint):
+    # See https://w3c.github.io/webdriver-bidi/#ref-for-remote-end-subscribe-steps%E2%91%A1.
+    top_level_context = await bidi_session.browsing_context.create(type_hint=type_hint)
+
+    on_entry = wait_for_event(CONTEXT_CREATED_EVENT)
+    await subscribe_events([CONTEXT_CREATED_EVENT])
+    context_info = await wait_for_future_safe(on_entry)
+
+    assert_browsing_context(
+        context_info,
+        top_level_context["context"],
+        children=None,
+        url="about:blank",
+        parent=None,
+        user_context="default"
+    )
```

Comment:
We haven't implemented this in Firefox yet, so I can't test it. But at this point we should have at least 2 contexts: one for the usual top_context, and one for the context created in the test.

Meaning we should at least get 2 events, and I'm not sure the spec is clear about which event should come first. It sounds fragile to assume the first event will be for `top_level_context`?

I imagine we can leverage https://github.com/web-platform-tests/wpt/issues/44702 once it lands. 
In the meantime, maybe just collect events in an array, wait for 0.5 seconds and check that we have at least one event for top_level_context?

Reply:
Oh right I didn't notice that! Sorry for the noise.

Reply:
Ah, no, you were totally correct. I have added it address your comment.

Reply:
Just a note that we might want to have a test for each a new tab, window, and (nested?) frame at the same time. But maybe we can do that once the PR @juliandescottes mentioned landed?